Apsitvarkom.IntegrationTests/DataAccess/DatabaseTests.cs

```diff
@@ -73,4 +73,17 @@ public async Task GetByPropertyTest()
             Assert.That(response.Notes, Is.EqualTo(dbRow.Notes));
         });
     }
+
+    [Test]
+    public async Task InsertAsync_SamePrimaryKeys_Throws()
+    {
+        // Try to insert the same row that was inserted in [SetUp]
+        var dbRow = DbInitializer.FakePollutedLocations.Value.Skip(3).Take(1).Single();
+
+        // Use a clean instance of the context to run the test
+        await using var context = new PollutedLocationContext(_options);
+        var dbRepository = new PollutedLocationDatabaseRepository(context);
+
+        Assert.ThrowsAsync<ArgumentException>(() => _ = dbRepository.InsertAsync(dbRow));
```

Comment:
```suggestion
        Assert.ThrowsAsync<ArgumentException>(() => dbRepository.InsertAsync(dbRow));
```

I guess that's not a mistake, but any reason for this?

Reply:
It's usually used to tell the reader of the code that the result of this method is discarded/not needed. I usually keep these, but since we don't return anything from `InsertAsync` we can remove it